.gitignore

```
@@ -128,3 +128,5 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+Koala.db
```

more database names are used
```suggestion
*.db
```